### ISSUE TITLE
RHEL distributions support

### ISSUE THREAD

**Issue by Fodoj** [hashicorp/otto#372]
Seems like currently Otto is more Ubuntu-focused. Is there a plan for adding support for RHEL-based distributions?

**Comment by mitchellh** [comment]
Yes, we actually plan on focusing only on Ubuntu and RHEL (or RHEL-like, Cent, Fedora, etc.) distributions. We're doing Ubuntu first and then RHEL will come after and we'll likely stop there for quite awhile.
